packages/@tinacms/fields/src/plugins/ListFieldPlugin.tsx

```diff
@@ -0,0 +1,438 @@
+/**
+
+Copyright 2019 Forestry.io Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+import * as React from 'react'
+import { Field, Form } from '@tinacms/forms'
+import styled, { css } from 'styled-components'
+import { FieldsBuilder } from '@tinacms/form-builder'
+import { IconButton } from '@tinacms/styles'
+import { Droppable, Draggable } from 'react-beautiful-dnd'
+import {
+  AddIcon,
+  DragIcon,
+  ReorderIcon,
+  TrashIcon,
+  LeftArrowIcon,
+} from '@tinacms/icons'
+import { GroupPanel, PanelHeader, PanelBody } from './GroupFieldPlugin'
+import { useFormPortal } from '@tinacms/react-forms'
+import { FieldDescription } from './wrapFieldWithMeta'
+
+interface ListFieldDefinititon extends Field {
+  component: 'list'
+  defaultItem?: object | (() => object)
```

Comment:
I wonder if the default item should be a string or another primitive.

Reply:
Yeah that's a mistake on my part, copying and pasting laziness 🤦